skbio/stats/composition.py

```diff
@@ -1105,6 +1211,64 @@ def _gram_schmidt_basis(n):
     return basis.T
 
 
+def _sbp_basis(sbp):
```

Comment:
We'll also want to include a unittest for this.  For starters, we can assert that the `_check_orthogonality` error never gets raised.